### PR TITLE
adding a second function to decrease even values.

### PR DESCRIPTION
**Pull request by AnthonySparling**
Just trying the pull request. This will set all even values to 2 but in a roundabout way.

### CODE REVIEW

File: process_data.py [hunk 1]
```diff
@@ -8,9 +8,13 @@ def process_value(x):
     if (5 * x % 2) == 1:
         return 5 * x + 1
     else:
-        return 5 * x
+        return extra_process(x)
     
-
+# Second function for additional process
+def extra_process(x):
```
Comment (adesautels-steer) [inline process_data.py]: Let's give this function a more descriptive name (mostly just showing you how inline comments work).

**Review by adesautels-steer — CHANGES_REQUESTED** [review]
Looks pretty good! Can you please address the one inline comment I made? I'll pull it in after that. Thanks!

**Comment by AnthonySparling** [comment]
Okay, function name has been updated and I had to correct an issue in the function.

**Comment by adesautels-steer** [comment]
Looks good—just resolved a conflict with Luc's latest commit and pulled it in!

**Pull request merged**